### DISCUSSION TITLE
Doubt related with the pathname of my model output file

### DISCUSSION THREAD

**Discussion by gonzaloalen** [usgs/pestpp#251] [General]
Hi all,

I am trying to use the pestpp-ies with my model. My model output file is not located in the working folder. If my working folder is:
      D:\...\Event\Landro.gid
My model output file is named "Hydrology_Hydrographs.csv" and is located here:
      D:\...\Event\Landro.gid\Time series\Hydrology_Hydrographs.csv

PEST++ Manual (section 4.14 - version 5.2.3) states that:
_The “model output” section of the PEST control file contains instruction file - model output file pairs. If a filename contains a space, its name must be enclosed in quotes. Pathnames must be provided for both instruction files and model output files if they do not reside in the current working folder._

So I defined my pst file as:
![image](https://user-images.githubusercontent.com/56552014/235549518-13209bac-9ccd-4d52-bdd1-f49a79890bdc.png)

However, even though I have included the quotes, it seems that pestpp-ies.exe is not able to read the path because of the spaces in the name of my "time series" folder:
![image](https://user-images.githubusercontent.com/56552014/235549992-f0eb9cc0-5a25-41c2-8206-ccfadb06d255.png)

The same document executed with pest.exe does not give me any problems. What mistake could I be making?

**Comment by briochh** [comment]
Hi @gonzaloalen
Not 100% sure but you going try just putting the quotations around just the “time series” bit.

**Reply by gonzaloalen** [reply]
Thank you for your answer but no, it is not solved by changing the quotations.

**Comment by jtwhite79** [comment]
The way the lines in the control file are being parsed is by whitespace so any spaces in paths like "Time Series" is going to throw things off.  Can you just rename "Time Series" to "Time_Series" in your workflow?    If not, I can try to modify the control file reader to support this...I don't want to tell you how to do your work, but I avoid spaces in paths like the plague because it causes all sorts of trouble...

**Reply by gonzaloalen** [reply]
As a user of the model, I am unable to change the name of the folder where the results are written. The model executable writes the results to this folder by default. I agree with you that the name of this folder should be corrected. I have contacted the developer of the model and he confirmed that the name of this folder will be edited in the next version.

Thank you, Jeremy. I don't believe there is a need to modify the code for my case, as I can wait for the new version of the model to be released.
